GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Parts/_ReportStatusBadge.cshtml

```diff
@@ -1,10 +1,80 @@
 ﻿@model ReportStatusBadgeViewModel
+@using GenderPayGap.Core
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+
+@{
+    Func<object, object> html = null;
+    string htmlClass = "";
+
+
+    switch(Model.ReportStatus)
+    {
+        case ReportStatusBadgeType.Due:
+            if (Model.Desktop)
+            {
+                html = @<text>DUE @Model.DateText.ToUpper()</text>;

```

Comment:
`<text>` isn't a real HTML tag as far as I know. Is there a particular reason you're using it here?

Reply:
It's essentially just being used for string interpolation here, we couldn't give these badges text as the formatting was incorrect and the classes are baked into the library component.